edx-platform/pearson-regis-theme/lms/templates/footer.html

```diff
@@ -66,46 +49,21 @@ <h4>Empowering people to make progress in their lives through learning</h4>
         </ul>
       </div>
       <div class="col-md-4 col-12 mid-footer-container">
-        <h4>Students</h4>
-        <p><a href="https://www.pearson.com/pathways/learner-profile.html/account">My Learner Profile</a></p>
-        <p><a href="https://www.pearson.com/pathways/recommended-programs.html">Get Recommendations</a></p>
-        <p><a href="https://www.pearson.com/pathways/areas-work-study.html">View Areas of Study</a></p>
-        <p><a href="https://www.pearson.com/pathways/program-profiles.html">Degrees, Courses &amp; Certificates</a></p>
-        <p><a href="https://www.pearson.com/pathways/student-resources.html">Student Resources</a></p>
-        <p><a href="https://www.pearson.com/pathways/blog.html">Education news and trends</a></p>
+        <p><a href="https://online.regiscollege.edu/accreditation/">Accreditation</a></p>
```

Comment:
I see the social media links open in a new tab, what do you think about doing the same for these links?. @alexjmpb